.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,77 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "bug: <title>"
+labels: [bug]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      examples:
+        - **Browser**: Firefox 89.0.2
+        - **OS**: Ubuntu 20.04
+        - **Node**: 14.17.0
+        - **npm**: 6.14.3
+    value: |
+        - Browser:
+        - OS: 
+        - Node: 
+        - npm: 
+    render: markdown
+  validations:
+    required: false
+- type: textarea
+  id: schema
+  attributes:
+    label: Your GraphQL Schema
```

Comment:
Maybe this should just be "Your Schema" ?

Reply:
```suggestion
    label: Your Schema
```